src/LinuxTracing/UprobesUnwindingVisitorDynamicInstrumentationTest.cpp

```diff
@@ -48,8 +48,13 @@ class UprobesUnwindingVisitorDynamicInstrumentationTest : public ::testing::Test
 
 TEST_F(UprobesUnwindingVisitorDynamicInstrumentationTest,
        VisitDynamicInstrumentationPerfEventsInVariousCombinationsSendsFunctionCalls) {
+  constexpr pid_t kPidTargetNamespace = 1042;
+  constexpr pid_t kTidTargetNamespace = 1043;
   constexpr pid_t kPid = 42;
   constexpr pid_t kTid = 43;
+  visitor_.SetInitialTidMapping({{kPidTargetNamespace, kPid}, {kTidTargetNamespace, kTid}});
```

Comment:
Two things:
- I would suggest bringing the additions you made to this test into a separate test.
- You are not testing the visits of `CloneExitPerfEventData` and `TaskNewtaskPerfEventData`, which is an important part of this change--you could add that to the separate test just mentioned.